src/i18n/en-US/index.js

```diff
@@ -51,6 +51,19 @@ export default {
       title: 'Libraries',
     },
   },
+  LibraryPage: {
+    text: {
+      title: '{name}',
```

Comment:
not useful if it's just a variable with no additional text ?

Reply:
Not agree, because it's translation.